src/ui/navigation/AppNavigator.tsx

```diff
@@ -250,7 +250,7 @@ function AppNavigator() {
     <AppStack.Navigator screenOptions={overlayScreenOptions}>
       {isPnPromptNeeded ? (
         <AppStack.Screen name={routeKeys.permissionGrantingPromptScreen}>
-          {() => <PermissionGrantingPrompt skipPnPermission={skipPnPermission} />}
+          {() => <PermissionGrantingPrompt skipPnPermission={skipPnPermission} allowSkip={true} />}
```

Comment:
Let's remove the `allowSkip` prop from here and set the default value to `true` in the component.